btr-web/btr-main-app/components/individual-person/AddNew.vue

```diff
@@ -142,7 +142,7 @@
           id="addNewPersonBirthdate"
           class="mt-3"
           :max-date="new Date()"
-          :placeholder="$t('placeholders.dateSelect.birthdate')"
+          :placeholder="significantIndividual.profile.birthDate || $t('placeholders.dateSelect.birthdate')"
```

Comment:
this should set the `initialDate` prop with the existing birthdate value instead of the placeholder